src/profession_group.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#ifndef CATA_SRCPROFESSION_GROUP_H
+#define CATA_SRCPROFESSION_GROUP_H
+
+#include "type_id.h"
+#include "json.h"
+
+struct profession_group {
+
+        static void load_profession_group( const JsonObject &jo, const std::string &src );
+        void load( const JsonObject &jo, const std::string & );
+        static const std::vector<profession_group> &get_all();
+        static void check_profession_group_consistency();
+        bool was_loaded;
```

Comment:
Is this used?

Reply:
Yes, it's part of the template magic that loads stuff from json